src/components/walking-distance/WalkingDistance.tsx

```diff
@@ -4,33 +4,29 @@ import {ThemeIcon} from '@atb/components/theme-icon';
 import {Walk} from '@atb/assets/svg/mono-icons/transportation';
 import {ThemeText} from '@atb/components/text';
 import {useHumanizeDistance} from '@atb/utils/location';
-import {StyleSheet, useTheme} from '@atb/theme';
+import {useTheme} from '@atb/theme';
 
 type Props = {
   distance: number | undefined;
   style?: StyleProp<ViewStyle>;
+  iconStyle?: StyleProp<ViewStyle>;
```

Comment:
Do we really need this?  Maybe ask the designers. And if we really need the difference, I don't think we want reusable components to expose fields for internal styling. It would then be better with something like `gapSize: 'small' | 'large'`. But all in all I think this should be avoidable.

Reply:
Reverted the change and made a new component for mobility instead